Products/Zuul/facades/eventclassesfacade.py

```diff
@@ -163,6 +163,36 @@ def getTransformTree(self, uid):
         })
         return transforms
 
+    def getTransformState(self, uid):
```

Comment:
API nitpick:  This should be `isTransformEnabled` since _transformEnabled_ is a boolean flag.